src/Raven.Client/Documents/Commands/CreateSubscriptionCommand.cs

```diff
@@ -15,18 +15,29 @@ public class CreateSubscriptionCommand : RavenCommand<CreateSubscriptionResult>,
     {
         private readonly SubscriptionCreationOptions _options;
         private readonly string _id;
+        private readonly bool _disabled;
 
         public CreateSubscriptionCommand(DocumentConventions conventions, SubscriptionCreationOptions options, string id = null)
         {
             _options = options ?? throw new ArgumentNullException(nameof(options));
             _id = id;
+            _disabled = false;
         }
+        public CreateSubscriptionCommand(DocumentConventions conventions, SubscriptionCreationOptions options, bool disabled, string id = null)
+        {
+            _options = options ?? throw new ArgumentNullException(nameof(options));
+            _id = id;
+            _disabled = disabled;
+        }
+       
 
         public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
         {
             url = $"{node.Url}/databases/{node.Database}/subscriptions";
+            url += "?disabled=" + Uri.EscapeDataString(_disabled.ToString());
```

Comment:
no need to escape?

Reply:
there should be no escaping here.

Reply:
But can only add that if it is et to true, no?